### PR TITLE
[RO-69] Move healthcheck endpoint to API root

### PR DESCRIPTION
**Pull request by Skyld**
Ticket: https://oat-sa.atlassian.net/browse/RO-69

Scope of this ticket is to move healthcheck endpoint to API root.

What was also done is an update on the readme file and on devops documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@
 - Application namespace has been changed from `App\` to `OAT\SimpleRoster\`.
 - Changed `APP_ROUTE_PREFIX` variable to exclude API version from it. Corresponding changes made to the `routes.yaml`/`security.yaml`
 - Renamed `UpdateLtiOutcomeAction` to `UpdateLti1p1OutcomeAction` for consistency.
+- Moved health check endpoint to API root.
```
Comment (uncleempty) [inline CHANGELOG.md:38]: a small one, could you please provide the endpoint itself here?

**Review by uncleempty — APPROVED** [review]
Approved, many thanks! Just a small question/request:
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Feature is working correctly on my local machine
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes
- [x] Documentation is updated according to changes

**Review by andreluizmachado — DISMISSED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes (if applicable)
- [x] Documentation is updated according to changes (if applicable)

**Review by uncleempty — APPROVED** [review]
Approved, :+1: !
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Feature is working correctly on my local machine
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes
- [x] Documentation is updated according to changes

**Review by andreluizmachado — APPROVED** [review]

**Pull request merged**